### ISSUE TITLE
BrowserModule is re-imported

### ISSUE THREAD

**Issue by Ghostbird** [fegyi001/mangol#31]
When I try to use Mangol in a lazy loaded module I run into this issue:
```log
ERROR Error: Uncaught (in promise): Error: BrowserModule has already been loaded.
If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.
```
The re-import is non-fatal when it does not happen in a lazy loaded module. However, in this case `<mangol>` cannot be used in the _template_. of a lazy loaded module component. So as a workaround:

1. Move the import to the root module of the app.
2. Add the MangolComponent as entryComponent to the root module of the app.
3. In the constructor of the component where you need the MangolComponent: Dynamically create a new MangolComponent using `ComponentFactoryResolver` and add it to the `ViewContainerRef` of the component.

However, this is quite an ugly workaround, as far as I know, non-root Angular modules should never import the BrowserModule.

**Comment by Cosmocat** [comment]
> However, this is quite an ugly workaround, as far as I know, non-root Angular modules should never import the BrowserModule.

@fegyi001 BrowserModule, BrowserAnimationsModule need to be removed from mangol.module.ts imports

**Comment by fegyi001** [comment]
@Ghostbird thanks for the issue, @Cosmocat thanks for the fix. New version will be out soon.

**Comment by fegyi001** [comment]
Fixed in 9.1.10-0.2
